src/StdUtils.sol

```diff
@@ -18,10 +18,12 @@ abstract contract StdUtils {
     address private constant CONSOLE2_ADDRESS = 0x000000000000000000636F6e736F6c652e6c6f67;
     uint256 private constant INT256_MIN_ABS =
         57896044618658097711785492504343953926634992332820282019728792003956564819968;
+    uint40 internal constant MAX_UNIX_TIMESTAMP = 2_147_483_647; // 2^31 - 1
```

Comment:
Hm why is this one relevant for solidity devs? I'd think UINT32_MAX would be more appropriate, since I don't think may devs are storing timestamps as signed ints (this value equals `type(int32).max`)